automation/jenkins_build.sh

```diff
@@ -365,6 +377,9 @@ EOSU
 }
 
 # Deploy
+# Test deploy
+deploy_resinhup_to_registries
```

Comment:
Why is this done here without any check? This same function is also called bellow in a check if the deploy flag is set